### PR TITLE
Add Tekton Triggers v0.6.0 Release

### PR DESCRIPTION
**Pull request by khrm**

### CODE REVIEW

**Comment by savitaashture** [comment]
Do we need to consider jobs for 4.3, 4.4 and 4.5
or is it just 4.5?

File: core-services/image-mirroring/tekton/mapping_tekton_triggers_v0_6_quay [hunk 1]
```diff
@@ -0,0 +1,3 @@
+registry.svc.ci.openshift.org/openshift/tektoncd-triggers-v0.6.0:tektoncd-triggers-controller quay.io/openshift-pipeline/tektoncd-triggers-controller:v0.6.0
+registry.svc.ci.openshift.org/openshift/tektoncd-triggers-v0.6.0:tektoncd-triggers-webhook quay.io/openshift-pipeline/tektoncd-triggers-webhoo:v0.6.0
```
Comment (vdemeester) [inline core-services/image-mirroring/tekton/mapping_tekton_triggers_v0_6_quay:2]: typo: `quay.io/openshift-pipeline/tektoncd-triggers-webhook:v0.6.0`

**Comment by khrm** [comment]
I added all three openshift version.
/hold cancel

**Comment by khrm** [comment]
@vdemeester @savitaashture Check this now.

**Pull request merged**